### PR TITLE
feat: ask for db root pass

### PR DESCRIPTION
**Pull request by alexiakl**
Ask for DB root password during installation with default being `root`.
Ask for DB IP during installation with default being `0.0.0.0`.
I still need to make sure nothing breaks on Linux.

### CODE REVIEW

**Comment by alexiakl** [comment]
@george-haddad is it easy for you to check this on Linux?

**Comment by george-haddad** [comment]
Right off the bat it works when I already have a MySQL installation on Ubuntu and I see the prompt for the password and default IP.

The only case that I see that needs tweaking is if I already have a MySQL or MariaDB installation located on another machine, vm, hosted else where ...etc. and I do not want the script to install a DB server without my specific authorization.

A nice tweak would be to prompt the user to use an existing DB installation (skip mysql install and go directly to ask for root pass and IP) or to install it on the spot, which will not prompt anything and use all defaults.

So these commands can be moved out of the install.sh and orchestrated from the `RoboFile.php`
```bash
 sudo debconf-set-selections <<< 'mysql-server mysql-server/root_password password root'
 sudo debconf-set-selections <<< 'mysql-server mysql-server/root_password_again password root'
 sudo apt-get -y install mysql-server
```

To recap.
1. Ask me if I want a new DB installed or use existing
1. If new then use all defaults and do not prompt me for anything
1. If existing then prompt me to enter my root password and IP of the server

**Comment by ginatrapani** [comment]
I pushed a commit to this branch that makes the prompt copy a little friendlier, as the target audience for this is beginners.

And yeah I like that suggestion @george-haddad!  It would be great to ask "Do you have an existing database you'd like to use and configure yourself? (y/n)" to start, and then follow on with these prompts if yes, and do all the defaults if no.

**Comment by alexiakl** [comment]
cool!

**Comment by alexiakl** [comment]
@ginatrapani @george-haddad 
I made some updates here, those need to be tested, especially on Ubuntu

**Comment by george-haddad** [comment]
@alexiakl I tested it on Ubuntu with various combinations, found some small Linux bugs which I committed the fix for. It seems to work on my end so it's good to go,
